src/types/ParticipantProject.ts

```diff
@@ -30,3 +31,16 @@ export type ParticipantProject = {
   votingDecision: VoteOption;
   whatNeedsToBeTrue: string;
 };
+
+export type ParticipantProjectSearchResult = {
+  cohortId: number;
+  cohortName: string;
+  country: string;
+  id: number;
+  landUseModelType: string;
```

Comment:
LandUseModel type is multi-select so string[]

Reply:
Thanks for the info, yes we can change this later as needed.
Can a participant project have more than one value for its land use type? If so, we need to change how it is rendered on the table.

Reply:
Yes it is a multi-select in the requirements: 

-Native Forest
-Monoculture
-Sustainable Timber
-Other Timber
-Mangroves
-Agroforestry
-Silvopasture
-Other Land-Use Model

Figma designs still need to be updated to reflect this I think.

Reply:
Got it, thanks @tommylau523 ... I can update the table later once the BE types are available.